sdk/remoterendering/azure-mixedreality-remoterendering/CHANGELOG.md

```diff
@@ -11,6 +11,16 @@
 ### Other Changes
 
 
+## 1.1.29 (2024-05-28)
+
+### Other Changes
+
+#### Dependency Updates
+
+- Upgraded `azure-mixedreality-authentication` from `1.2.22` to version `1.2.23`.
```

Comment:
Shouldn't this be `1.2.24`? That's the version of MR Auth we just released.

Reply:
Yeah. Need to fix script to do a better job for the same date release for parent-children dependency packages.